Assets/Scripts/Player/PlayerController.cs

```diff
@@ -9,6 +10,10 @@ public class PlayerController : MonoBehaviour {
     private int maxHealth = 100;
     [SerializeField]
     private int currentHealth;
+    [SerializeField]
+    private int healthDecayDamage = 5;
+    [SerializeField]
+    private float healthDecayTime = 5f;

```

Comment:
I think these values should be set in the Player prefab and not in this script. If the default values in this script are changed in the future, then those default value changes will not be applied to the Player prefab. It's better to just have one place where these values get set because there's no need to have default values for these variables.